### ISSUE TITLE
Pretrained model on Imagenet

### ISSUE THREAD

**Issue by ngocson1804** [zhoudaquan/rethinking_bottleneck_design#3]
Do you have pretrained model on imagenet? if yes, please share with me, thanks!

**Comment by zhoudaquan** [comment]
> Do you have pretrained model on imagenet? if yes, please share with me, thanks!

@ngocson1804 Yeah we will release the pretrained model once got approved form the company.

**Comment by mynameisliuliu** [comment]
hi，，in ssdlite-pytorch-mobilenext. The compression pack of pretrained models is damaged. I really need this weight pack，Please help me，Please！
